Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wrangler"
-version = "1.20.0"
+version = "1.20.1"
```

Comment:
Do we want to deprecate in a patch version?

Reply:
I agree, I'll open a new PR